include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -855,6 +925,38 @@ class FEEvaluationBase
    */
   unsigned int cell;
 
+  /**
+   * Flag holding information whether a face is on the left or the right of a
+   * cell. Not used for cells.
+   **/
+  bool is_interior_face;
+
+  /**
+   * Stores the index an FEFaceEvaluation object is currently pointing into
+   * (minus face, plus face, face associated with cell).
+   */
+  unsigned int face_vector_access_index;
+
+  /**
+   * Stores the current number of a face within the given cell in case
+   * `is_face==true`, using values between `0` and `2*dim`.
```

Comment:
> \<code\> is_face==true \</code\>  

and alike, i think.

Reply:
This should be one of a number of valid ways of typesetting code, so I guess that should be fine (at least from a technical point of view).

Reply:
> This should be one of a number of valid ways of typesetting code

probably, I just did not notice it elsewhere in deal.ii.

Reply:
There are maybe a few dozen places, e.g. here:
https://github.com/dealii/dealii/blob/master/include/deal.II/base/mpi.h#L392

Reply:
indeed. Wonder why do we bother with `<code>blabla</code>` elsehwere? 😄